src/fen_util.rs

```diff
@@ -47,6 +47,7 @@ pub fn place_pieces(chessboard: &mut Chessboard, fen_rows: &str) {
 /// # Returns
 ///
 /// A boolean indicating whether the FEN string is valid.
+#[allow(unused_variables)]
```

Comment:
@abrahimt what's this for?

Reply:
I think I put that during dev and forgot to take it out. It shouldn't be there.